kotlin-core/src/main/java/com/segment/analytics/kotlin/core/SegmentDestination.kt

```diff
@@ -19,14 +20,14 @@ import java.util.concurrent.atomic.AtomicInteger
 class SegmentDestination(
```

Comment:
<documentation> may be good to add here since it's such an integral part of our library